### ISSUE TITLE
Feedback bar fontsize/heading level

### ISSUE THREAD

**Issue by smcmurtry** [veteransaffairscanada/vac-benefits-directory#1417]
https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b758b92156c13b1cbd5a0a0

"Did you find the information you're looking for?" should be bigger (size 18px) and should be an h2 to make it easier to find with screen reader nav (this is what gov.uk does with their feedback bar: https://www.gov.uk/).

**Comment by SupeDeDupe** [comment]
The back button won't allow the user to navigate back to the home page.

**Comment by SupeDeDupe** [comment]
Other elements in the feedback bar do not match design specifications that should be adjusted:

https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b758b92621f406ad06fb4e3

https://app.zeplin.io/project/5acbd659dc553ec84805afd4/screen/5b758b93414d8f626ba8e19d
